api/handlers_association.go

```diff
@@ -0,0 +1,44 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/YaleSpinup/apierror"
+	"github.com/YaleSpinup/ec2-api/ssm"
+	"github.com/gorilla/mux"
+)
+
+func (s *server) DescribeAssociationHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
This should be in `handlers_instances.go`, as well as the other Command Handler, since they both apply to `instances`

Reply:
Done.